host/xtest/regression_1000.c

```diff
@@ -1473,3 +1473,63 @@ static void xtest_tee_test_1020(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1020, xtest_tee_test_1020,
 		"Test lockdep algorithm");
+
+static void xtest_tee_test_1021(ADBG_Case_t *c)
+{
+	TEEC_Session session = { 0 };
+	TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
+	uint32_t ret_orig;
+	uint8_t *empty_buf = NULL;
+
+	/*
+	 * First check if the test is relevant because this feature
+	 * may be not supported
+	 */
+	if (!xtest_teec_ctx.memref_null) {
```

Comment:
I'm not sure this is needed. All gits are supposed to be on latest when testing.

Reply:
@jenswi-linaro correct

Reply:
Sorry, I'm bit lost. I understood that we are supposed to support old drivers. Hence with a driver that doesn't enable this feature this test should not be run.

Reply:
No worries. From test point of view we're always assuming that the gits are updated and correctly configured.

The compatibility stuff is for third party code over which we have no control. It may be hard for some projects to do synchronized updates of everything (optee_client, optee_os and the kernel).

Reply:
ok removed